From 87107c18853ea955fb2bbce8bfedc74fe92d08ce Mon Sep 17 00:00:00 2001
From: Chris Lamb <lamby@debian.org>
Date: Fri, 24 Jul 2009 11:40:53 +0200
Subject: [PATCH] Install manpages under $(PREFIX)/share/man/, not $(PREFIX)/man/

The Filesystem Hierarchy Standard dictates that we should place
manpages under 'share', not under the prefix directly.

Signed-off-by: Chris Lamb <lamby@debian.org>
---
 libfiu/Makefile      |    4 ++--
 preload/run/Makefile |    4 ++--
 utils/Makefile       |    6 +++---
 3 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/libfiu/Makefile b/libfiu/Makefile
index f80b76c..f72c513 100644
--- a/libfiu/Makefile
+++ b/libfiu/Makefile
@@ -71,8 +71,8 @@ install-lib: libs libfiu.pc
 	@echo
 
 install-man:
-	$(INSTALL) -d $(PREFIX)/man/man3
-	$(INSTALL) -m 0644 libfiu.3 $(PREFIX)/man/man3/
+	$(INSTALL) -d $(PREFIX)/share/man/man3
+	$(INSTALL) -m 0644 libfiu.3 $(PREFIX)/share/man/man3/
 
 install: install-lib install-man
 
diff --git a/preload/run/Makefile b/preload/run/Makefile
index 4328548..f69e035 100644
--- a/preload/run/Makefile
+++ b/preload/run/Makefile
@@ -63,8 +63,8 @@ install: fiu_run_preload.so fiu-run
 	$(INSTALL) -m 0755 fiu_run_preload.so $(PREFIX)/lib
 	$(INSTALL) -d $(PREFIX)/bin
 	$(INSTALL) -m 0755 fiu-run $(PREFIX)/bin
-	$(INSTALL) -d $(PREFIX)/man/man1
-	$(INSTALL) -m 0644 fiu-run.1 $(PREFIX)/man/man1/
+	$(INSTALL) -d $(PREFIX)/share/man/man1
+	$(INSTALL) -m 0644 fiu-run.1 $(PREFIX)/share/man/man1/
 
 clean:
 	rm -f $(OBJS) fiu_run_preload.so fiu-run build-flags
diff --git a/utils/Makefile b/utils/Makefile
index 3943711..17bdde7 100644
--- a/utils/Makefile
+++ b/utils/Makefile
@@ -12,9 +12,9 @@ install:
 	$(INSTALL) -d $(PREFIX)/bin
 	$(INSTALL) -m 0755 fiu-ctrl $(PREFIX)/bin
 	$(INSTALL) -m 0755 fiu-ls $(PREFIX)/bin
-	$(INSTALL) -d $(PREFIX)/man/man1
-	$(INSTALL) -m 0644 fiu-ctrl.1 $(PREFIX)/man/man1/
-	$(INSTALL) -m 0644 fiu-ls.1 $(PREFIX)/man/man1/
+	$(INSTALL) -d $(PREFIX)/shared/man/man1
+	$(INSTALL) -m 0644 fiu-ctrl.1 $(PREFIX)/share/man/man1/
+	$(INSTALL) -m 0644 fiu-ls.1 $(PREFIX)/share/man/man1/
 
 clean:
 
-- 
1.6.3.3

